runtime/interpreter/conversion.go

```diff
@@ -85,6 +85,7 @@ func ByteValueToByte(element Value) (byte, error) {
 	return b, nil
 }
 
+// TODO: meter
```

Comment:
should be safe to remove this todo?

Reply:
There seem to be still some outstanding TODOs and PRs too:
- https://github.com/onflow/cadence/pull/1578
- https://github.com/onflow/cadence/pull/1598